docs/handbook/systems-administration.md

```diff
@@ -506,6 +506,35 @@ To revert your zone to proxy-less configuration, run
 # pkg set-property use-system-repo False
 ```
 
+### Troubleshooting
+
+Zone configuration and management operations can fail for a number of reasons, sometimes obscure.
+This section contains information on how to solve different well-known issues.
+
+#### Fixing zone installation issues
+
+Zones on OpenIndiana use ZFS features to manage boot environments.
+Zone's root and its parent directory should be a ZFS filesystem.
```

Comment:
This sentence puzzles me, that it's not enough for those two directories to be **on** ZFS filesystem, but that they themselves have to be ZFS filesystems. Sentence like this would enlighten it for me:
```suggestion
Zone's root and its parent directory should be a ZFS filesystem.
You can check that those two directories are ZFS filesystems by:
`# zfs list -t filesystem rpool/zones`
```
...or whichever directories we use on OI by default for zones.